content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -427,7 +427,44 @@ For a TCP probe, the kubelet makes the probe connection at the node, not in the
 means that you can not use a service name in the `host` parameter since the kubelet is unable
 to resolve it.
 
+### Setting TerminationGracePeriodOverride
 
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+Prior to release 1.21, the pod-level `terminationGracePeriodSeconds` was used
+for terminating a container that failed its liveness or startup probe.
+
+In 1.21, when the feature flag `ProbeTerminationGracePeriod` is enabled, a user
```

Comment:
please add `ProbeTerminationGracePeriod` to [Feature gates for Alpha or Beta features](https://github.com/kubernetes/website/blob/master/content/en/docs/reference/command-line-tools-reference/feature-gates.md#feature-gates-for-alpha-or-beta-features) and add a short description of the feature in the [List of feature gates](https://github.com/kubernetes/website/blob/master/content/en/docs/reference/command-line-tools-reference/feature-gates.md#list-of-feature-gates-feature-gates). thanks!